osquery/tables/events/tests/linux/process_events_tests.cpp

```diff
@@ -114,7 +114,13 @@ TEST_F(ProcessEventsTests, syscall_name_label) {
 TEST_F(ProcessEventsTests, exec_event_processing) {
   // clang-format off
   const RawAuditEvent kSampleExecveEvent = {
+#if defined(__x86_64__)
     { 1300, "audit(1502125323.756:6): arch=c000003e syscall=59 success=yes exit=0 a0=23eb8e0 a1=23ebbc0 a2=23c9860 a3=7ffe18d32ed0 items=2 ppid=6882 pid=7841 auid=1000 uid=1000 gid=1000 euid=1000 suid=1000 fsuid=1000 egid=1000 sgid=1000 fsgid=1000 tty=pts1 ses=2 comm=\"sh\" exe=\"/usr/bin/bash\" subj=unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023 key=(null)" },
+#elif defined(__aarch64__)
+    { 1300, "audit(1502125323.756:6): arch=c00000b7 syscall=221 success=yes exit=0 a0=23eb8e0 a1=23ebbc0 a2=23c9860 a3=7ffe18d32ed0 items=2 ppid=6882 pid=7841 auid=1000 uid=1000 gid=1000 euid=1000 suid=1000 fsuid=1000 egid=1000 sgid=1000 fsgid=1000 tty=pts1 ses=2 comm=\"sh\" exe=\"/usr/bin/bash\" subj=unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023 key=(null)" },
```

Comment:
Nitpick, this, and the below `exec_event_processing`, changes include a bit too much copy-paste for my liking. Do you mind pulling out the syscall change into something like:
```
#if ...
  const std::string kExecSyscall{"59"};
```
then interpolating/concatting here?

Reply:
Right, I'm not super fan of it either. The original had a completely separate audit string & parsing which was worse imho. I'll see what I can do. Ideally we should change "arch" as well though I think for the sake of this test our parser doesn't care.